src/styles/Book.css

```diff
@@ -0,0 +1,130 @@
+.book-row {
+  background-color: white;
+  margin: 3% 5%;
+  padding: 25px 25px;
+  border: 1px solid #eee;
+  display: flex;
+}
+
+.left-container {
+  width: 40%;
+  text-align: left;
+}
+
+p {
+  margin: 0;
+}
+
```

Comment:
- [ ] Yo've done really well with the design. However, styling should EXACTLY be the same as the one provided in the designs. Please see below, highlighted differences between the two designs. Especially, the user icon and the "Add New Book" section.

Your desig
![Screenshot 2021-09-16 at 21-01-37 React App](https://user-images.githubusercontent.com/70150423/133686346-4aa453cb-57f2-450f-871a-0e82c7d1d01c.png)

Design Template
![Screenshot_2021-09-16_21-03-37](https://user-images.githubusercontent.com/70150423/133686428-8d2dab8d-b373-4072-b8f2-090b61900873.png)